docs/content/guides/developer/app-examples/coin-flip.mdx

```diff
@@ -433,208 +453,297 @@ Now take a look at the main function in this game, `finish_game`:
+│  │ ObjectType: 0x2::coin::Coin<0x2::sui::SUI>                                                            │
+│  │ Version: 75261540                                                                                     │
+│  │ Digest: Ezmi94kWCfjRzgGTwnXehv9ipPvYQ7T6Z4wefPLRQPPY                                                  │
+│  └──                                                                                                     │
+│ Published Objects:                                                                                       │
+│  ┌──                                                                                                     │
+│  │ PackageID: 0x4120b39e5d94845aa539d4b830743a7433fd8511bdcf3841f98080080f327ca8                         │
+│  │ Version: 1                                                                                            │
+│  │ Digest: 5XbJkgx8RSccxaHoP3xinY2fMMhwKJ7qoWfp349cmZBg                                                  │
+│  │ Modules: counter_nft, house_data, single_player_satoshi                                               │
+│  └──                                                                                                     │
+╰──────────────────────────────────────────────────────────────────────────────────────────────────────────╯
 ```
 
-This represents a basic example of a coin flip backend in Move. The game module, `single_player_satoshi`, is prone to MEV attacks, but the user experience for the player is streamlined. Another example game module, `mev_attack_resistant_single_player_satoshi`, exists that is MEV-resistant, but has a slightly downgraded user experience (two player-transactions per game).
+Save the `PackageID` and the `ObjectID` of the `HouseCap` object you receive in your own response to [connect to your frontend](#connecting-your-package).
 
-You can read more about both versions of the game, and view the full source code for all the modules in the [Satoshi Coin Flip repository](https://github.com/MystenLabs/satoshi-coin-flip).
+In this case, the `PackageID` would be `0x4120b39e5d94845aa539d4b830743a7433fd8511bdcf3841f98080080f327ca8` and the `HouseCap` ID would be `0xfa1f6edad697afca055749fedbdee420b6cdba3edc2f7fd4927ed42f98a7e63a`.
 
-Now that you have written our contracts, it's time to deploy them.
+### Next steps
 
-## Deployment
+Well done. You have written and deployed the Move package! 🚀
 
-{@include: ../../../snippets/initialize-sui-client-cli.mdx}
+To turn this into a complete dApp, you need to [create a frontend](#frontend).
 
-{@include: ../../../snippets/publish-to-devnet-with-coins.mdx}
+## Frontend {#frontend}
 
-The package should successfully deploy. Now, it's time to create a frontend that can interact with it.
-
-## Frontend
+In this final part of the app example, you build a frontend (UI) that allows end users to place bets and take profits, and let's the admin manage the house.
```

Comment:
```suggestion
In this final part of the dApp example, you build a frontend (UI) that allows end users to place bets and take profits, and lets the admin manage the house.
```
dApp used elsewhere but I only mention it here because I had to get that apostrophe.